pom.xml

```diff
@@ -95,7 +106,7 @@
                 </configuration>
             </plugin>
 
-            <plugin>
+            <!--<plugin>
```

Comment:
I've found that jacoco can work here in caes of configuring it with the following:
`                <configuration>
                    <includes>
                        <include>org/togglz/**</include>
                    </includes>
                </configuration>
`

Reply:
Jacoco works again, just commented in in again. Was broken during migration, but forgot to re-enable it.